compiler/src/iree/compiler/Codegen/LLVMCPU/test/split_reduction_pipeline_tests.mlir

```diff
@@ -0,0 +1,348 @@
+// RUN: iree-opt --pass-pipeline='hal.executable(hal.executable.variant(iree-llvmcpu-lower-executable-target))' --split-input-file %s | FileCheck %s
```

Comment:
Ideally we should put the test to https://github.com/iree-org/iree/tree/main/compiler/src/iree/compiler/Codegen/Sandbox/test and test with the pass itself.

I'm fine with this because the tests are covered.